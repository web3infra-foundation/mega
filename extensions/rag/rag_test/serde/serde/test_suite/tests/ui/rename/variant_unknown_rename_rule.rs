use serde_derive::Serialize;

#[derive(Serialize)]
enum S {
    #[serde(rename_all = "abc")]
    V {
        name: u8,
        long_name: u8,
        very_long_name: u8,
    },
}

fn main() {}
