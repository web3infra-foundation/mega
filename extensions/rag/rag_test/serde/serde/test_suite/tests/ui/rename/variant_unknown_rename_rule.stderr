error: unknown rename rule `rename_all = "abc"`, expected one of "lowercase", "UPPERCASE", "PascalCase", "camelCase", "snake_case", "SCREAMING_SNAKE_CASE", "kebab-case", "SCREAMING-KEBAB-CASE"
 --> tests/ui/rename/variant_unknown_rename_rule.rs:5:26
  |
5 |     #[serde(rename_all = "abc")]
  |                          ^^^^^
